src/index.js

```diff
@@ -45,8 +45,9 @@ videojs.use('video/mux', (player) => {
         setupMuxDataTracking(player);
       }
 
-      setupSubtitlesForPlayer(player);
-      
+      if (typeof setupSubtitlesForPlayer !== 'undefined') {
```

Comment:
When VHS is used, `mux-subtitles` is excluded from the build and thus `setupSubtitlesForPlay` is undefined, do a null check here before calling it.